components/app/leaderboard/MobileProposalFigures.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="flex justify-between sm:justify-around">
+    <div class="rounded border border-darkgrey p-1 px-2">
+      <table>
+        <tbody>
+          <tr class="text-primary">
+            <td class="text-right sm:w-24">
+              {{ addPunctuation(proposal.yesVotes) }}
+            </td>
+            <td class="pl-1">{{ $t('leaderboard.proposal.votes.yes') }}</td>
+          </tr>
+          <tr>
+            <td class="text-right">{{ addPunctuation(proposal.noVotes) }}</td>
+            <td class="pl-1">{{ $t('leaderboard.proposal.votes.no') }}</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+
+    <div class="self-center">
```

Comment:
suggestion(blocking): i think this div can be omitted if the child div uses the `self-center` class as well